WordPress/src/main/java/org/wordpress/android/ui/posts/adapters/PostListAdapter.kt

```diff
@@ -101,7 +110,17 @@ class PostListAdapter(
         notifyDataSetChanged()
     }
 
-    private class LoadingViewHolder(view: View) : ViewHolder(view)
+    private class LoadingViewHolder(val view: View) : ViewHolder(view) {
+        fun onBind(item: LoadingItem) {
+            (view.findViewById(R.id.skeleton_button_edit) as View?)?.setVisible(item.options.showEditButton)
```

Comment:
NitPick: We might want to consider passing `uiHelpers` into the constructor and replacing 
`(view.findViewById(R.id.skeleton_button_edit) as View?)?.setVisible(item.options.showEditButton)`
with 
`uiHelpers.updateVisibility(view.findViewById(R.id.skeleton_button_edit), item.options.showEditButton)` to keep the view holder consistent with the other view holders.

We could even store the view into a field so we don't invoke `findViewById` whenever the view gets recycled.

Wdyt?

Reply:
Fixed in e293df4bf575a17bf8a2d909a54fd173e58bd3e9